src/Microsoft.DocAsCode.MarkdigEngine.Extensions/Microsoft.DocAsCode.MarkdigEngine.Extensions.csproj

```diff
@@ -10,4 +10,10 @@
     <ProjectReference Include="..\Microsoft.DocAsCode.MarkdigEngine.Validators\Microsoft.DocAsCode.MarkdigEngine.Validators.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Reference Include="System.ComponentModel.DataAnnotations">
+      <HintPath>..\..\..\..\..\..\..\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.6.1\System.ComponentModel.DataAnnotations.dll</HintPath>
```

Comment:
Suggest to simply validate using code. <Reference> like this won't work in .net core.